pom.xml

```diff
@@ -65,14 +65,14 @@
     <version.antapacheregexp>1.8.2</version.antapacheregexp>
     <version.org.reflections>0.9.11</version.org.reflections>
     <version.javax.inject>1</version.javax.inject>
-    
-    <version.org.infinispan>10.0.0.Beta3</version.org.infinispan>
-    <version.org.infinispan.protostream>4.3.0.Alpha6</version.org.infinispan.protostream>
+
+    <version.org.infinispan>10.0.0.Beta5</version.org.infinispan>
```

Comment:
Why not [10.0.0.CR1](https://mvnrepository.com/artifact/org.infinispan/infinispan-core/10.0.0.CR1)? This is the version I've been using with the operator tests.

Reply:
I guess the idea is just to align with whatever Quarkus is using

Reply:
that's correct, it is to align with what quarkus currently define.